### ISSUE TITLE
Exeption in 181.0.4

### ISSUE THREAD

**Issue by axute** [zielu/GitToolBox#84]
I get this error in 181.0.4

```
Argument for @NotNull parameter 'project' of zielu/gittoolbox/util/GtUtil.getCurrentRepositoryQuick must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'project' of zielu/gittoolbox/util/GtUtil.getCurrentRepositoryQuick must not be null
	at zielu.gittoolbox.util.GtUtil.$$$reportNull$$$0(GtUtil.java)
	at zielu.gittoolbox.util.GtUtil.getCurrentRepositoryQuick(GtUtil.java)
	at zielu.gittoolbox.ui.statusbar.GitStatusWidget.lambda$onCacheChange$1(GitStatusWidget.java:64)
	at com.intellij.openapi.application.TransactionGuardImpl$2.run(TransactionGuardImpl.java:315)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:447)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:431)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:415)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.n(IdeEventQueue.java:779)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:720)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:395)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by zielu** [comment]
Could you give more details ? 
What IDE are you using and what kind of actions were you trying to perform.

**Comment by zielu** [comment]
Note to self: this could happen after project is closed and GitStatusWidget is disposed. There should be additional check for isDisposed(). Also messageBus connection should be disconnected in dispose (super implementation only nulls it out)

**Comment by axute** [comment]
I so the update and restart Phpstorm.

**Comment by zielu** [comment]
Fixed in 181.0.5
